ratarmount.py

```diff
@@ -271,7 +272,7 @@ def __init__(
             SQLiteIndexedTar._openCompressedFile( fileObject, gzipSeekPointSpacing, encoding )
 
         # will be used for storing indexes if current path is read-only
-        possibleIndexFilePaths = [
+        possibleIndexFilePaths = [indexFileName] if indexFileName is not None else [
```

Comment:
`[ os.path.abspath( indexFileName ) ]` would be better, because the check whether the index path is writable on line 300 expects an absolute path implicitly.